### PR TITLE
fix all the Arachnid crap I've been putting off

### PR DESCRIPTION
**Pull request by Maldaris**
## About The Pull Request

There's a handful of things about Arachnids that aren't really up to snuff with the principle about non-human races. Namely, Arachnids are OP in a few key areas. This hopefully addresses that.

## Why It's Good For The Game

Arachnids should be a fun alternative, but not a nominally better roundstart race.

## Changelog
:cl:
add: Arachnid mutation toxin recipe: 1 Unstable Mutation Toxin, 10u Heparin.
balance: Initiating cocooning costs less nutrients
balance: Arachnid night vision halved strength
balance: Arachnids can still eat but don't like Gross food.
balance: Arachnid Cocoons take 2/3rds less time to break out of.
fix: Spider meat from gibbed Arachnids now uses the spider meat icon.
fix: Cocooning consumes nutrient after the cocooning is complete.
/:cl:

### CODE REVIEW

**Comment by SynnGraffkin** [comment]
Define gross food, in regards to giant spiders.

**Comment by Maldaris** [comment]
> Define gross food, in regards to giant spiders.

Quite literally, anything with the `GROSS` attribute

**Comment by SynnGraffkin** [comment]
Does raw meat count as "gross" in regards to a giant spider? Considering that Lizards are willing to eat raw meat, including rats and raw organs, I would assume spiders do too. Also, weren't you gonna touch up the resist timer for cocoons?

**Comment by Maldaris** [comment]
>Also, weren't you gonna touch up the resist timer for cocoons?
Ah right.

**Comment by tiramisuapimancer** [comment]
GROSS food tag is different from RAW- GROSS is stuff like soggy tofu, dessicated candy corn, and burnt messes. Lizards and deviant tastes humans will love eating these. RAW is raw meats.

**Comment by SynnGraffkin** [comment]
So wouldn't spiders eat raw meat then, Mald?

File: waspstation/code/modules/mob/living/carbon/human/species_types/spider.dm [hunk 1]
```diff
@@ -51,8 +52,8 @@ GLOBAL_LIST_INIT(spider_last, world.file2list("strings/names/spider_last.txt"))
 	attack_sound = 'sound/weapons/slash.ogg'
 	miss_sound = 'sound/weapons/slashmiss.ogg'
 	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/spider
-	liked_food = MEAT | RAW | GROSS
-	disliked_food = FRUIT
+	liked_food = MEAT | RAW
+	disliked_food = FRUIT | GROSS
```
Comment (Maldaris) [inline waspstation/code/modules/mob/living/carbon/human/species_types/spider.dm:56]: @SynnGraffkin

**Comment by Superyodeler** [comment]
Since both uses of `spinner_rate` have multipliers, it would probably be a good idea to change spinner rate itself.

**Comment by Maldaris** [comment]
> Since both uses of `spinner_rate` have multipliers, it would probably be a good idea to change spinner rate itself.

`spinner_rate` gets used with web placing and the metric is it costs 3x as much nutrition to cocoon someone/something compared to just laying a web on a tile.

File: waspstation/code/game/objects/effects/spiderperson_web.dm [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
     icon = 'icons/effects/effects.dmi'
-    icon_state = "stickyweb1"
     desc = "It's stringy and sticky, but the threads are larger than what spiderlings could produce."
     anchored = TRUE
     density = FALSE
     max_integrity = 20
 
+/obj/structure/spider_player/New()
+	icon_state = pick(list("stickyweb1", "stickyweb2"))
+
```
Comment (Superyodeler) [inline waspstation/code/game/objects/effects/spiderperson_web.dm:11]: This needs to be initialized. It currently leaves the structure with a null integrity value.
```suggestion
/obj/structure/spider_player/New()
	..()
	icon_state = pick(list("stickyweb1", "stickyweb2"))


```

File: waspstation/code/game/objects/effects/spiderperson_web.dm [hunk 3]
```diff
@@ -52,7 +54,7 @@
 	. = ..()
 
 /obj/structure/spider_player/cocoon/container_resist(mob/living/user)
-	var/breakout_time = 3000 // DECI not DECA ffs
+	var/breakout_time = 1000 // DECI not DECA ffs
```
Comment (Superyodeler) [inline waspstation/code/game/objects/effects/spiderperson_web.dm:57]: Should mention this in the changelog.

**Pull request merged**